tests/DbQueryModuleTest.php

```
@@ -249,6 +255,44 @@ public function testSelectPagerEntity(): void
+
+        $page = $list[1];
+        assert($page instanceof Page);
+        assert(is_array($page->data));
+
+        $this->assertInstanceOf(TodoConstruct::class, $page->data[0]);
+        $this->assertSame('RUN', $page->data[0]->title);
+        $factoryInstances = TodoInjectionFactory::$instances;
+        $this->assertGreaterThan(0, $factoryInstances);
+
+        $page = $list[1];
+        assert($page instanceof Page);
```

PHPStan (2.1.54)

[error] Call to function assert() with true will always evaluate to true.
Because the type is coming from a PHPDoc, you can turn off this check by setting treatPhpDocTypesAsCertain: false in your %configurationFile%.

(function.alreadyNarrowedType)